cfgov/v1/models/browse_filterable_page.py

```diff
@@ -22,6 +22,9 @@
 from v1.models.learn_page import EventPage
 
 
+NEWSROOM_CACHE_TAG = "newsroom"
```

Comment:
If this were something like the newsroom page slug (or any filterable list's page slug), could this be implemented on the filterable list itself, instead of being specific to Newsroom? We already have logic [in `signals.py`](https://github.com/cfpb/consumerfinance.gov/pull/6781/files#diff-79d70b0e1eabb1410a6294959f01356ac3d11035c190a72b93f0e512e2032116R111-R121) to determine whether a page belongs to a filterable list.

Reply:
I think the options for cache-tag use could grow a lot once we get access to Akamai configs and to the API cache_tag invalidation request. Newsroom seems to be the most pressing case, and I'd be thrilled to get that far.

Reply:
Now that Akamai config has been ironed out, I'd propose merging this to fix newsroom caching, and then moving on to other targets, such as the landing page at `/about-us/blog/`, which could get a cache-tag purge whenever a BlogPage or LegacyBlogPage is published.

Since we control how cache-tags are applied, with no Akamai configs needed, the newsroom tag won't lock us in to any scheme if we want to use namespaces or concoct a wider filterable-list cache group.

Simple tags makes sense to me for targeted purges of special pages like `about-us/blog/` and `about-us/newsroom/`